### PR TITLE
Handling error message mongoDB

### PR DESCRIPTION
**Pull request by tungnt-580**

### CODE REVIEW

File: config/database.js [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+})
+
+db.on('disconnected', () => {
+  console.log('MongoDB connection disconnected')
+})
 
-// eslint-disable-next-line no-console
-db.on('error', console.error.bind(console, 'MongoDB connection error:'))
+db.on('error', (err) => {
+  console.log('MongoDB connection error. Please try to start/restart your MongoDB')
```
Comment (kevcomedia) [inline config/database.js]: Could you change this to `console.error`? Let's direct error messages to proper channels

**Review by kevcomedia — CHANGES_REQUESTED** [review]

File: config/database.js [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
+
+mongoose.connect(dbURI)
 
 mongoose.Promise = Promise
 
 const db = mongoose.connection
 
-// Bind connection to error event (to get notification of connection errors)
+db.on('connected', () => {
+  console.log('MongoDB connection connected to ' + dbURI)
```
Comment (kevcomedia) [inline config/database.js:13]: Our linter is not so happy with us using `console.log`. For now disable the `no-console` rule for this file, just so it shuts up.

Maybe later we can consider using a proper logging library

File: config/database.js [hunk 3]
```diff
@@ -1,12 +1,22 @@
 const mongoose = require('mongoose')
 
-mongoose.connect('mongodb://127.0.0.1/WeAreYourTeam')
+const dbURI = 'mongodb://127.0.0.1/WeAreYourTeam' //should put it to .env
```
Comment (kevcomedia) [inline config/database.js]: Please add a space after the `//`

**Review by kevcomedia — CHANGES_REQUESTED** [review]
Make linter happy

File: config/database.js (hunk 2 above)
Comment (sachamorgese) [reply config/database.js:13]: Yeah, honestly we can disable the rule all together on the back-end. I just copied what we use at my current workplace (which is one big repository for both front and back-end) but in this case, logging isn't a big deal.

Comment (tungnt-580) [reply config/database.js:13]: Added disable no-console in the top of file

**Review by kevcomedia — APPROVED** [review]

**Pull request merged**